modules/gapi/include/opencv2/gapi/infer/parsers.hpp

```diff
@@ -64,14 +64,15 @@ detection is smaller than confidence threshold, detection is rejected.
 given label will get to the output.
 @return a tuple with a vector of detected boxes and a vector of appropriate labels.
 */
-GAPI_EXPORTS std::tuple<GArray<Rect>, GArray<int>> parseSSD(const GMat& in,
-                                                            const GOpaque<Size>& inSz,
-                                                            const float confidenceThreshold = 0.5f,
-                                                            const int   filterLabel = -1);
+GAPI_EXPORTS std::tuple<GArray<Rect>, GArray<int>> parseSSDFBL(const GMat& in,
+                                                               const GOpaque<Size>& inSz,
+                                                               const float confidenceThreshold = 0.5f,
+                                                               const int   filterLabel = -1);
```

Comment:
Is `filterLabel` used to filter background class id? Why not to use corresponding field from `DetectionOutput` layer? (named `background_label_id`)

Reply:
This parameter provides id of label (is `DetectionOutput layer`[1]) which needs to skip.